### PR TITLE
Update to 0.9.3

### PR DESCRIPTION
**Pull request by lorepirri**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -22,14 +22,16 @@ requirements:
...
-    - importlib_metadata  # [py<38]
+    - python
     - numpy
-    - pandas >=1.2.0
-    - pyarrow
+    - pandas >=1.3.5
     - six
+    - pyarrow >=6.0.1
+    - fsspec >=2022.5.0
+    # TODO: remove this hard dependency
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
```
This is not a comment for us, please remove.

**Review by JeanChristopheMorinPerso — APPROVED** [review]
Please use the default PR template when creating a PR.

**Comment by boldorider4** [comment]
Links in description?

**Review by boldorider4 — APPROVED** [review]
LGTM

**Pull request merged**